src/Exception/ApiException.php

```diff
@@ -3,6 +3,35 @@
 
 /**
  */
-class ApiException extends BunqException
+class ApiException extends \Exception
 {
+    /**
+     * The first item index in an array.
+     */
+    const INDEX_FIRST = 0;
+
+    /**
+     * @var int
+     */
+    private $responseCode;
+
+    /**
+     * ApiException constructor.
+     *
+     * @param int $responseCode
+     */
+    public function __construct($message, $args = [])
```

Comment:
Please add strict types for arguments (PHP 7.0 style) and doc block params. Also, why are we using `$args`? Is it necessary here? If so, please add a comment explaining this to the constructor header.